### ISSUE TITLE
Add a changelog

### ISSUE THREAD

**Issue by durdenx** [whitecolor/ts-node-dev#66]
Is there a changelog somewhere?

**Comment by whitecolor** [comment]
No, because the target behavior of the library is not a subject for a change. -)

**Comment by jeremyben** [comment]
No changelog is fine for such a small scope, but versionning is weird I must say : `1.0.0-pre.X`
And the npm package version is not on par with the repo package.json.
Right now, it's `1.0.0-pre.39` on npm and `1.0.0-pre.34` in the package.json.

The `yarn publish` command allows you to bump the package.json version before publishing on npm and adds a tagged commit, so you could actually have the git history as a changelog.

**Comment by whitecolor** [comment]
Because it is a kind of pre-release, because I wasn't sure if this thing going to work out -) Ok, I will probably remove pre.

**Comment by jeremyben** [comment]
You're close to 40 000 downloads a week. I guess it's working out 😉

**Comment by numfin** [comment]
why not using np? Its very easy
